### PR TITLE
Enable GPPC tests in ICW

### PR DESCRIPTION
**Pull request by adam8157**
Enable existing GPPC tests and move more from TINC to ICW to get higher
coverage.

For now we keep gppc_test, gppc_demo and tabfunc_gppc_demo test suites
separate, may investigate more and merge them.

Signed-off-by: Adam Lee <ali@pivotal.io>
Signed-off-by: Yuan Zhao <yuzhao@pivotal.io>

### CODE REVIEW

**Comment by ivannovick** [comment]
this is cool!

File: src/interfaces/gppc/Makefile [hunk 1]
```diff
@@ -2,7 +2,6 @@
 #
 # Makefile for src/backend/gppc library
```
Comment (danielgustafsson) [inline src/interfaces/gppc/Makefile]: Not a fault of this patch but might as well fix this comment to match the actual pathname while at it. The same incorrect pathname is in `src/interfaces/gppc/test/Makefile`

File: src/interfaces/gppc/Makefile [hunk 2]
```diff
@@ -41,7 +39,9 @@ installcheck: install
 
 uninstall: uninstall-lib
 
-clean distclean: clean-lib
+clean:
 	rm -f $(OBJS)
  
+distclean: clean clean-lib
+
```
Comment (danielgustafsson) [inline src/interfaces/gppc/Makefile]: Why this change?

**Comment by adam8157** [comment]
Also found tons of GPPC cases in tinc, going to move them to ICW.

File: src/interfaces/gppc/Makefile (hunk 2 above)
Comment (adam8157) [inline src/interfaces/gppc/Makefile]: `distclean` is supposed to clean more than `clean`, I just see and change it. The original behavior is OK with me too :)
